### ISSUE TITLE
Enable Wayland support

### ISSUE THREAD

**Issue by travier** [flathub/org.rncbc.qpwgraph#14]
Tracking issue for enabling Wayland support for this app. See previous discussion and issues blocking this in https://github.com/flathub/org.rncbc.qpwgraph/issues/10.

**Comment by travier** [comment]
🎉 Thanks!

**Comment by travier** [comment]
Re-opening as we're reverting the change. See https://github.com/flathub/org.rncbc.qpwgraph/pull/19 & https://github.com/flathub/org.rncbc.qpwgraph/issues/18

**Comment by boredsquirrel** [comment]
uhm, what is the current status of this?

For me it works fine, the global menu is the issue? after adding another task bar, and adding the "global menu" applet to it (Fedora Kinoite 40) the global menu didnt display for any app, not kate, not dolphin etc.

but qpwgraph works how I would expect it to, but only after applying this override

```
[Context]
sockets=!x11;wayland;

[Environment]
QT_QPA_PLATFORM=wayland
```

**Comment by rncbc** [comment]
may I ask why the upstream recommended platform is not to stay put as is?

you probably (don't) know that is being hardcoded/enforced to `QT_QPA_PLATFORM=xcb` by now, unless you build it with `cmake -DCONFIG_WAYLAND=ON`...

anyway, there's more than just the system-tray icon and main menus that fail to work right on wayland, so please stick with what works and "looks" the same for everybody else, not only flatpak'ers ;)

**Comment by rncbc** [comment]
as of v0.9.6 - Get rid of CONFIG_WAYLAND build config option.
